core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/recipes/RecipeV4Endpoint.java

```diff
@@ -11,55 +19,45 @@
 import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import java.util.Set;
 
-import com.sequenceiq.cloudbreak.api.model.RecipeRequest;
-import com.sequenceiq.cloudbreak.api.model.RecipeResponse;
-import com.sequenceiq.cloudbreak.api.model.RecipeViewResponse;
-import com.sequenceiq.cloudbreak.doc.ContentType;
-import com.sequenceiq.cloudbreak.doc.ControllerDescription;
-import com.sequenceiq.cloudbreak.doc.Notes;
-import com.sequenceiq.cloudbreak.doc.OperationDescriptions.RecipeOpDescription;
-
-import io.swagger.annotations.Api;
-import io.swagger.annotations.ApiOperation;
-
-@Path("/v3/{workspaceId}/recipes")
+@Path("/v4/{workspaceId}/recipes")
 @Consumes(MediaType.APPLICATION_JSON)
-@Api(value = "/v3/{workspaceId}/recipes", description = ControllerDescription.RECIPE_V3_DESCRIPTION, protocols = "http,https")
-public interface RecipeV3Endpoint {
+@Api(value = "/v4/{workspaceId}/recipes", description = ControllerDescription.RECIPES_V4_DESCRIPTION, protocols = "http,https")
+public interface RecipeV4Endpoint {
 
     @GET
     @Path("")
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = RecipeOpDescription.LIST_BY_WORKSPACE, produces = ContentType.JSON, notes = Notes.RECIPE_NOTES,
             nickname = "listRecipesByWorkspace")
-    Set<RecipeViewResponse> listByWorkspace(@PathParam("workspaceId") Long workspaceId);
+    Set<RecipeV4ViewResponse> list(@PathParam("workspaceId") Long workspaceId);
```

Comment:
so we can use interable object as a return type?
@horadla23 told me that this is not the way what we'd like to follow

Reply:
No, we can't use. @doktoric please change it as well